pkgs/osu-lazer-bin/update.sh

```diff
@@ -6,8 +6,11 @@ oldversion=$(jq -r '.version' "$info")
 url="https://github.com/ppy/osu/releases/download/${version}/osu.AppImage"
 
 if [ "$oldversion" != "$version" ]; then
-  nix store prefetch-file "$url" --json > "$info"
-  jq --arg version "$version" '.version = $version' "$info" | sponge "$info"
+  if nix store prefetch-file "$url" --json >"$info"; then
+    jq --arg version "$version" '.version = $version' "$info" | sponge "$info"
```

Comment:
This will still cause `info.json` to be empty if the AppImage isn't available (tested by giving a bogus URL since `osu-lazer` has now released the AppImage). The `>"$info"` occurs no matter the exit status of `nix store prefetch-file`.

Maybe something like:
```suggestion
  if output=$(nix store prefetch-file "$url" --json); then
    jq --arg version "$version" '.version = $version' <<< "$output" > "$info"
```
This would eliminate the need for `sponge` and consequently `moreutils` as well.

Reply:
Thanks!